xyz

```diff
@@ -111,7 +111,7 @@ case "$increment" in
   *) echo "Invalid --increment" >&2 ; exit 1 ;;
 esac
 
-[[ $(git rev-parse --abbrev-ref HEAD) == $branch ]] ||
```

Comment:
```bash
[[ $(git rev-parse --abbrev-ref HEAD) == $branch ]] ||
                                         ^-- SC2053
```

[SC2053][1]: Quote the rhs of `==` in `[[ ]]` to prevent glob matching.


[1]: https://github.com/koalaman/shellcheck/wiki/SC2053